UIKitTemplate/UIKitTemplate/FeedScreen/StoryViewCell.swift

```diff
@@ -0,0 +1,172 @@
+// StoryViewCell.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// Ячейка сториз
+final class StoryViewCell: UITableViewCell {
+    // MARK: - Constants
+
+    enum Constants {
+        static let yourStoryText = "Ваша история"
+        static let storyWithSpasingWeight = 86
+    }
+
+    // MARK: - Visual Components
+
+    private let profilePhoto: UIImageView = {
+        let imageView = UIImageView()
+        imageView.contentMode = .scaleAspectFill
+        imageView.translatesAutoresizingMaskIntoConstraints = false
+        imageView.layer.cornerRadius = 30
+        imageView.clipsToBounds = true
+        return imageView
+    }()
+
+    private let yourStoryLabel: UILabel = {
+        let label = UILabel()
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.text = Constants.yourStoryText
+        label.font = .init(name: "Verdana", size: 8)
+        label.textAlignment = .center
+        label.textColor = .lightGray
+        return label
+    }()
+
+    private let addStoryButtont: UIButton = {
+        let button = UIButton()
+        button.translatesAutoresizingMaskIntoConstraints = false
+        let plusImageView = UIImageView()
+        plusImageView.image = .addButton
+        plusImageView.translatesAutoresizingMaskIntoConstraints = false
+        button.addSubview(plusImageView)
+        plusImageView.centerXAnchor.constraint(equalTo: button.centerXAnchor).isActive = true
+        plusImageView.centerYAnchor.constraint(equalTo: button.centerYAnchor).isActive = true
+        plusImageView.heightAnchor.constraint(equalToConstant: 10).isActive = false
+        plusImageView.widthAnchor.constraint(equalToConstant: 10).isActive = false
+        button.backgroundColor = .pinkAddButton
+        button.layer.cornerRadius = 10
+        return button
+    }()
+
+    private let scrollView = UIScrollView()
+
+    // MARK: - Initializers
+
+    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+        createView()
+        setConstraints()
+    }
+
+    required init?(coder: NSCoder) {
+        super.init(coder: coder)
+    }
+
+    // MARK: - Public Methods
+
+    func loadUserPhoto(_ photo: UIImage) {
+        profilePhoto.image = photo
+    }
+
+    func loadStories(_ storiesMap: [ViewingStatus: [Story]]) {
+        let numberOfStories = (storiesMap[.isViewed]?.count ?? 0) + (storiesMap[.isNotViewed]?.count ?? 0)
```

Comment:
```suggestion
       loadStories
```
разнеси по методам эту функцию такую кашу нельзя оставлять